baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -196,19 +186,15 @@ Only allow this for allowlisted domains SHALL be enabled.
 
 To configure the settings for External Chat:
 
-#### Policy Group 4 Common Instructions
+#### GWS.CHAT.4.1v0.1 Instructions
 1.  Sign in to the [Google Admin Console](https://admin.google.com).
 2.  Select **Apps** -\> **Google Workspace** -\> **Google Chat**.
 3.  Select **External Chat Settings** -\> **Chat externally**.
-
-#### GWS.CHAT.4.1v0.1 Instructions
-1.  Select **ON**.
-2.  Select **Save**.
-
-#### GWS.CHAT.4.2v0.1 Instructions
-1.  Select **Only allow this for allowlisted domains**
-2.  To add allowlisted domains select **Manage allowlisted domains**
-3.  Select **Save**.
+4.  Select **OFF** OR.
+5.  Select **ON** AND.
+6.  Select **Only allow this for allowlisted domains**.
+7.  To add allowlisted domains select **Manage allowlisted domains**.
+8.  Select **Save**.
```

Comment:
I think the OR and AND instructions might be confusing. What if you did something like this:

```
To enable external chat for allowlisted domains only:
1. Sign in to the [Google Admin Console](https://admin.google.com).
2. Select **Apps** -\> **Google Workspace** -\> **Google Chat**.
3. Select **External Chat Settings** -\> **Chat externally**.
4. Select **ON**
5. Select **Only allow this for allowlisted domains**.
6. To add allowlisted domains select **Manage allowlisted domains**.
7. Select **Save**.

Alternatively, to disable external chat entirely:
1. Sign in to the [Google Admin Console](https://admin.google.com).
2. Select **Apps** -\> **Google Workspace** -\> **Google Chat**.
3. Select **External Chat Settings** -\> **Chat externally**.
4. Select **OFF**
5. Select **Save**.

Reply:
@jkaufman-mitre Any update on this?

Reply:
Adding this here for ease of tracking:

In the baseline, the criticality should be changed to 'Should' for 4.1, since whether the setting 'External Chat Sharing' is 'ON' / 'OFF' it is a 'pass' in the both cases, with a restriction that if 'ON' allow only trusted domains.